### PR TITLE
Papers: NetPaxos, Paxos Made Switch-y, P4FPGA, and Consensus in  NVM

### PR DESCRIPTION
**Pull request by tudang**

### CODE REVIEW

File: _data/publications.yml [hunk 1]
```diff
@@ -13,7 +13,32 @@
     date: '2015-05-04'
     url: 'https://www.usenix.org/system/files/conference/nsdi15/nsdi15-paper-jose.pdf'
     type: 'conference'
--    
+
+-
+    title: "NetPaxos: consensus at network speed"
```
Comment (jnfoster) [inline _data/publications.yml]: Please capitalize the title exactly as in the published version?

File: _data/publications.yml [hunk 2]
```diff
@@ -36,3 +61,11 @@
     date: '2018-08-20'
     url: 'https://conferences.sigcomm.org/sigcomm/2018/workshop-netcompute.html'
     type: 'workshop'
+
+-
+    title: "Consensus for Non-Volatile Main Memory"
+    authors: "Huynh Tu Dang, Jaco Hofmann, Yang Liu, Marjan Radi, Dejan Vucinic, Fernando Pedone, and Robert Soulé"
+    venue: "IEEE ICNP European P4 Workshop"
```
Comment (jnfoster) [inline _data/publications.yml]: Is this the name of the workshop?

File: _data/publications.yml [hunk 3]
```diff
@@ -36,3 +61,11 @@
...
+    date: '2018-09-24'
+    url: 'https://p4.org/events/2018-09-24-p4-eu-workshop/'
```
Comment (jnfoster) [inline _data/publications.yml]: I'd rather wait to add this entry until the PDF is published at an official URL.

**Comment by tudang** [comment]
Hi Nate, I pushed a new commit to address the issues you mentioned.

**Comment by jnfoster** [comment]
Thanks!

**Pull request merged**